CefSharp.Core/Internals/CefFrameWrapper.cpp

```diff
@@ -234,10 +234,24 @@ Int64 CefFrameWrapper::Identifier::get()
 IFrame^ CefFrameWrapper::Parent::get()
 {
     ThrowIfDisposed();
-    auto parent = _frame->GetParent();
-    if (parent != nullptr)
+    if (_parentFrame != nullptr)
```

Comment:
Yeah, I've cleaned it up locally to look more like the IBrowser returning method in this class.